### PR TITLE
Add new sync indicator view

### PR DESCRIPTION
**Pull request by msasikanth**
- Rename `SyncIndicatorView` to `SyncIndicatorView_Old`
- Duplicate `SyncIndicatorView_Old` to create `SyncIndicatorView`
- Extend `ConstraintLayout` for `SyncIndicatorView`
- Add new sync icon drawables
- Add new sync indicator view
- Use new sync indicator view layout in `SyncIndicatorView`
- Add new `SyncIndicatorView` to patients screen
- Changing spacing in `view_simple_video` layout
- Remove old sync indicator view
- Add lint rule to ignore long vector path in `ic_syncing`
- Remove unused resources
- Update CHANGELOG

**Story**: https://app.clubhouse.io/simpledotorg/story/4220/add-new-sync-indicator-view

### CODE REVIEW

File: app/src/main/java/org/simple/clinic/sync/indicator/SyncIndicatorView.kt [hunk 1]
```diff
@@ -67,17 +67,17 @@ class SyncIndicatorView(
...
-    get() = binding.rootLayout
+    get() = binding.root
+
+  private val statusImageView
+    get() = binding.statusImageView
 
   private val statusTextView
     get() = binding.statusTextView
 
-  @SuppressLint("CheckResult")
```
Comment (Honey14) [inline app/src/main/java/org/simple/clinic/sync/indicator/SyncIndicatorView.kt:80]: Why did we remove this?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -413,7 +413,7 @@
   <!--- Sync Indicator-->
   <string name="syncindicator_status_pending">Sync pending</string>
   <string name="syncindicator_status_failed">Sync failed</string>
-  <string name="syncindicator_status_syncing">Syncing…</string>
```
Comment (Honey14) [inline app/src/main/res/values/strings.xml:416]: The design still has three dots when syncing, did we remove this by mistake?

**Review by Honey14 — CHANGES_REQUESTED** [review]
LGTM. Left 2 comments. PTAL?

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (msasikanth) [reply app/src/main/res/values/strings.xml:416]: These are the latest designs: https://www.figma.com/file/vmZlmlFSCOrEYzTMkqsNB8/Simple-Production?node-id=18287%3A32415

File: app/src/main/java/org/simple/clinic/sync/indicator/SyncIndicatorView.kt (hunk 1 above)
Comment (msasikanth) [reply app/src/main/java/org/simple/clinic/sync/indicator/SyncIndicatorView.kt:80]: We don't have any results of Rx stream subscriptions inside `onFinishInflate`, so we don't need it anymore.

**Review by Honey14 — APPROVED** [review]
LGTM

**Pull request merged**